### PR TITLE
Fix cases in 4.x

### PR DESCRIPTION
**Pull request by wewang58**
@openshift/devexp-qe pls review it, thx

### CODE REVIEW

File: features/cli/build.feature [hunk 1]
```diff
@@ -23,9 +23,9 @@ Feature: build 'apps' with CLI
     Then the output should contain:
       | ruby-hello-world |
     When I run the :new_build client command with:
-      | app_repo |  openshift/ruby:2.0~https://github.com/openshift/ruby-hello-world.git |
-      | strategy | docker                                                                |
-      | name     | n1                                                                    |
+      | app_repo |  openshift/ruby:latest~https://github.com/openshift/ruby-hello-world.git |
+      | strategy | docker                                                                   |
```
Comment (xiuwang) [inline features/cli/build.feature]: This docker build will not fail?

Comment (wewang58) [reply features/cli/build.feature]: not fail

Comment (akostadinov) [reply features/cli/build.feature]: lgtm when @xiuwang is happy

Comment (xiuwang) [reply features/cli/build.feature]: LGTM

File: features/cli/build.feature [hunk 2]
```diff
@@ -23,6 +23,9 @@ Feature: build 'apps' with CLI
     Then the output should contain:
       | ruby-hello-world |
     When I run the :new_build client command with:
+      | app_repo |  openshift/ruby:latest~https://github.com/openshift/ruby-hello-world.git |
+      | strategy | docker                                                                   |
+      | name     | n1                                                                       |
```
Comment (wzheng1) [inline features/cli/build.feature]: I saw above 3 lines are duplicated with beleow 3 lines?

File: features/cli/build.feature (hunk 1 above)
Comment (wzheng1) [reply features/cli/build.feature]: Build not fail? use ruby:25 with ruby-hello-world to do docker build?

File: features/cli/build.feature (hunk 2 above)
Comment (wewang58) [reply features/cli/build.feature]: Updated, I think now you are happy.

File: features/cli/build.feature (hunk 1 above)
Comment (akostadinov) [reply features/cli/build.feature]: seems like this comment needs addressing

File: features/cli/build.feature (hunk 2 above)
Comment (akostadinov) [reply features/cli/build.feature]: @wewang58 , does PR look good for you now.

File: features/cli/build.feature (hunk 1 above)
Comment (wzheng1) [reply features/cli/build.feature]: yes, this needs to be addressed.

Comment (wzheng1) [reply features/cli/build.feature]: This has been addressed.

File: features/cli/build.feature (hunk 2 above)
Comment (wzheng1) [reply features/cli/build.feature]: yes, LGTM

**Pull request merged**